fs-add/etc/init.d/dhcpc-event

```diff
@@ -45,10 +45,6 @@ renew|bound)
   for i in $dns ; do
     echo "nameserver $i" >> $RESOLV_CONF
   done
-  # kill always ntpd
-  if test -r /var/run/ntpd.pid; then
-    kill -9 $(cat /var/run/ntpd.pid) 2> /dev/null
-  fi
```

Comment:
This must be the reason why `ntpd` inevitable quits after some time

Reply:
@mark-royds I take it you're using static IP addresses and not DHCP since this doesn't seem to affect you?

Reply:
> @mark-royds I take it you're using static IP addresses and not DHCP since this doesn't seem to affect you?

It is DHCP but with a reservation if that makes any difference

Reply:
Okay, strange. It shouldn't make any difference in this case. Nevertheless, it's no longer important since this code is removed.